excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreatorTest.java

```diff
@@ -481,11 +481,11 @@ public void caseTypeUserRoleEnttyInvalidCrud_createErrorMessageCalled_defaultMes
     }
 
     @Test
-    public void caseFieldEntityInvalidUserRole_createErrorMessageCalled_customMessageReturned_whenIdamRoleMissing() {
+    public void caseFieldEntityInvalidUserRole_createErrorMessageCalled_customMessageReturned_whenIdamAPMissing() {
```

Comment:
Rename test when validation error `CaseFieldEntityInvalidUserRoleValidationError` is renamed

Reply:
Fixed in 3951e2b